app/lib/common/widgets/chat/convo_with_avatar_card.dart

```diff
@@ -1,12 +1,16 @@
 import 'package:acter/common/providers/chat_providers.dart';
+import 'package:acter/common/providers/room_providers.dart';
+import 'package:acter/common/themes/app_theme.dart';
 import 'package:acter/common/themes/colors/color_scheme.dart';
 import 'package:acter/features/chat/providers/chat_providers.dart';
 import 'package:acter/features/chat/widgets/room_avatar.dart';
 import 'package:acter_avatar/acter_avatar.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:skeletonizer/skeletonizer.dart';
 
 class ConvoWithAvatarInfoCard extends ConsumerWidget {
+  final String? title;
```

Comment:
where are using this down in a widget??? seems not mentioned anywhere...

Reply:
yeah, no that was an in-between attempt that got abandoned ...